app/views/transcribe/_save_buttons.html.slim

```diff
@@ -2,27 +2,24 @@
   -unless @preview_xml
     -if @page && !@page.ai_plaintext.blank? && @collection.text_entry?
       =button_tag t('.copy_ai_plaintext'), type: 'button', id: 'copy-ai-plaintext', title: "#{t('.copy_ai_plaintext_tooltip')}"
-    =button_tag t('.preview'), :name => 'preview', type: 'submit', title: "#{t('.preview_tooltip')}"
+    =button_tag t('.preview'), name: 'preview', type: 'submit', title: "#{t('.preview_tooltip')}"
   -else
     =button_tag t('.edit'), :name => 'edit', type: 'submit', title: "#{t('.edit_tooltip')}"
   -unless @collection.subjects_disabled || @collection.field_based
-    =button_tag t('.autolink'), :name => 'autolink', type: 'submit', title: "#{t('.autolink_tooltip')}"
+    =button_tag t('.autolink'), name: 'autolink', type: 'submit', title: "#{t('.autolink_tooltip')}"
 
   / check status of page and configuration of collection
   -if @page.status.blank? || @page.status == Page::STATUS_INCOMPLETE || @page.status == Page::STATUS_BLANK
-    =button_tag t('.save_changes'), :name => 'save_to_incomplete', type: 'submit', id: 'save_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.save_to_incomplete_tooltip')}", data: { disable_with: t('.please_wait') }
+    =button_tag t('.save_changes'), name: 'save_to_incomplete', type: 'submit', id: 'save_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.save_to_incomplete_tooltip')}", data: { disable_with: t('.please_wait') }, onclick: 'unsavedNotes(event);'
 
   -if [Page::STATUS_NEEDS_REVIEW, Page::STATUS_INDEXED, Page::STATUS_TRANSCRIBED].include?(@page.status)
-    -if @collection.review_workflow
-      =button_tag t('.save_changes'), :name => 'save_to_needs_review', type: 'submit', id: 'save_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.save_to_needs_review_tooltip')}", data: { disable_with: t('.please_wait') }
-    -else
-      =button_tag t('.save_changes'), :name => 'save_to_transcribed', type: 'submit', id: 'save_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.save_to_transcribed_tooltip')}", class: "bggreen", data: { disable_with: t('.please_wait') }
+    =button_tag t('.save_changes'), name: 'save_to_needs_review', type: 'submit', id: 'save_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.save_to_needs_review_tooltip')}", data: { disable_with: t('.please_wait') }
 
-  -if [nil, "", Page::STATUS_INCOMPLETE].include?(@page.status)
-    -if @collection.review_workflow
-      =button_tag t('.done'), :name => 'save_to_needs_review', type: 'submit', id: 'finish_button_top', onclick: 'unsavedNotes(event);', title: "#{t('.finish_to_needs_review_tooltip')}", class: "bggreen", data: { disable_with: t('.please_wait') }
+  -if [nil, "", Page::STATUS_NEEDS_REVIEW, Page::STATUS_INCOMPLETE].include?(@page.status)
+    -if @collection.review_workflow && @page.status != Page::STATUS_NEEDS_REVIEW
```

Comment:
Based on descriptions, I don't think we need a separate 'review_incomplete' status. We just have to double check that if review_workflow == true && status != review for the condition here